static/css/page-user.less

```diff
@@ -104,6 +104,16 @@ div#subjectsPage{
   }
 }
 
+div#lists{
+  columns: 1;
```

Comment:
If you use columns you shouldn't use the float:left above.

Also you'll probably need a fixed height.

Have you considered a flex box solution for this? That's probably going to be the best result here.

Reply:
@jdlrobson I have made some changes.
![1](https://user-images.githubusercontent.com/64412143/103967721-927d0100-5188-11eb-8498-d33c77ca512d.JPG)